### PR TITLE
[Fix] json.loads() error caused by different response type of http.client and httplib

### PR DESCRIPTION
**Pull request by Xuanwo**
`httplib`'s `HTTPResponse.read()` returns `str`, while `http.client`'s `HTTPResponse.read()` returns `bytes`, which caused json.loads() error.

This bug can be reproduced on any version on python3, reproduced code here:

```python
import qingcloud.qingstor

conn = qingcloud.qingstor.connect(
        'pek3a.qingstor.com',
        'abc',
        'def'
    )

bucket = conn.get_bucket('xyz')
print(bucket.stats())
```

This bug can pass unit test normally.

### CODE REVIEW

File: qingcloud/qingstor/bucket.py [hunk 1]
```diff
@@ -67,13 +78,13 @@ def get_key(self, key_name, validate=True):
...
             err.message = "Request not authenticated, Access Key ID is either " \
-                "missing or invalid."
+                          "missing or invalid."
             raise err
         elif response.status == 403:
             err = get_response_error(response)
             err.code = "permission_denied"
             err.message = "You don't have enough permission to accomplish " \
-                "this request."
+                          "this request."
```
Comment (osier) [inline qingcloud/qingstor/bucket.py]: unrelated change

File: qingcloud/qingstor/bucket.py [hunk 2]
```diff
@@ -46,6 +45,18 @@ def __getitem__(self, key_name):
     def __len__(self):
         pass
 
+    def load_data(self, response_data):
```
Comment (osier) [inline qingcloud/qingstor/bucket.py]: The issue should be existing across the sdk, not only QingStor Bucket.

File: qingcloud/qingstor/bucket.py [hunk 3]
```diff
@@ -46,6 +45,18 @@ def __getitem__(self, key_name):
...
+        """ Judge the type of data, fix the json.loads error.
```
Comment (osier) [inline qingcloud/qingstor/bucket.py]: The docstring looks unrelated with the function's purpose. 

"""Wrapper to load json data, to be compatible with Python3
....
...
"""

File: qingcloud/qingstor/bucket.py (hunk 2 above)
Comment (Xuanwo) [inline qingcloud/qingstor/bucket.py]: I tested qingcloud iaas wihtout problem, just happen on qingstor.
